WSL/install.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Install WSL
+description: Install Windows Subsystem for Linux with the command, wsl --install. Use a Bash terminal on your Windows machine run by your preferred Linux distribution - Ubuntu, Debian, SUSE, Kali, Fedora, Pengwin, Alpine, and more are available.
+keywords: wsl, install, BashOnWindows, bash, windows subsystem for linux, install ubuntu on windows, enable WSL2, linux on windows
+ms.date: 09/10/2021
+ms.topic: article
+ms.localizationpriority: high
+---
+
+# Install WSL
+
+## Prerequisites
+
+You must be running Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11.
+  
+> [!NOTE]
+> To check your Windows version and build number, select **Windows logo key + R**, type **winver**, select **OK**. You can [update to the latest Windows version](ms-settings:windowsupdate) in the Settings menu or using the [Windows Update Assistant](https://www.microsoft.com/software-download/windows10).
+
+If you're running an older build, or just prefer not to use the install command and would like step-by-step directions, see **[WSL manual installation steps for older versions](./install-manual.md)**.
+
+## Install
+
+You can now install everything you need to run Windows Subsystem for Linux (WSL) by entering this command in PowerShell or Windows Command Prompt and then restarting your machine.
+
+```powershell
+wsl --install
+```
+
+This command will enable the required optional components, download the latest Linux kernel, set WSL 2 as your default, and install a Linux distribution for you *(Ubuntu by default, see below to change this)*.
+
+The first time you launch a newly installed Linux distribution, a console window will open and you'll be asked to wait for files to de-compress and be stored on your machine. All future launches should take less than a second.
+
+## Change the default Linux distribution installed
+
+By default, the installed Linux distribution will be Ubuntu. This can be changed using the `-d` flag.
+
+- To change the distribution installed, enter: `wsl --install -d <Distribution Name>`. Replace `<Distribution Name>` with the name of the distribution you would like to install.
+- To see a list of available Linux distributions available for download through the online store, enter: `wsl --list --online` or `wsl -l -o`.
+- To install additional Linux distributions after the initial install, you may also use the command: `wsl --install -d <Distribution Name>`.
+
+> [!TIP]
+> If you want to install additional distributions from inside a Linux/Bash command line (rather than from PowerShell or Command Prompt), you must use .exe in the command: `wsl.exe --install -d <Distribution Name>` or to list available distributions: `wsl.exe -l -o`.
+
+If you run into an issue during the install process, check the [installation section of the troubleshooting guide](./troubleshooting.md#installation-issues).
+
+## Set up your Linux user info
```

Comment:
Suggestion: Can we add a link to the best practices doc somewhere here? I would think we'd want to get them started on looking at that doc as the "Next Steps" right after they install so they can see all the cool things they can do with WSL.

Reply:
Great suggestion! Done. 💯 👍